### DISCUSSION TITLE
Switching on/off the video on the camera.

### DISCUSSION THREAD

**Discussion by rikrdo89** [roleoroleo/yi-hack-Allwinner-v2#697] [Q&A]
I know there is a toggle in the UI to switch the video on/off, but Is there a command or a script in the camera that I can use to switch off the video of the camera on a schedule (say from midnight to 5am) via CRON?

**Comment by roleoroleo** [comment]
Yes.
`ipc_cmd -t off`
and
`ipc_cmd -t on`
